ubuntuStation/forms.py

```diff
@@ -25,18 +25,8 @@ def __call__(self, form, field):
             #print("Error")
             raise ValidationError('Entry with this value already exists in the database')
 
-'''
 class QuestionForm(FlaskForm):
     questiontext = TextAreaField('questiontext', validators=[DataRequired()], description=u'Question')
-    questionurl = TextField('questionurl', validators=[DataRequired()], description=u'Question URL')
-'''
-'''
-class ActiveQuestionForm(FlaskForm):
```

Comment:
@jygh98 this class being removed is associated to the other commit no? Notice you removed the class ActiveQuestion information from the other files in the other commit. So I will have to refuse the request again unless I am overlooking something.